README.md

```diff
@@ -165,7 +161,7 @@ curl base_url/user/award/ \
 
 **Request :**
 ```
-curl base_url/user/<username> -X GET
+curl base_url/users/<sample_user_name> -X GET
```

Comment:
Question @roymoran and @omkarjc27 should we use `id` or `username` as the identifier? Does it matter?

Reply:
I guess username will be more easy for users to remember. I suggest **we should not** have an auto generated id for users. We should only generate the API key for communities.